subt/docker/cudatest/rospywrapper.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python2
```

Comment:
What is this test about? Does it test CUDA, or ROS. If ROS, should it be in this changelist, or even in the cudatest image in the long run?

Reply:
> What is this test about?

Well... It is complicated.

In order to run ok on cloudsim, the solution must call `/subt/start` and /subt/finish` service. If the solution does not call these, cloudsim sees it as an error and restarts the whole simulation. The last time I tried this, it kept going for several days in this vicious cycle, the end result being "error" on the webpage and no logfiles to download.

So the point of this file is to "do the dance" cloudsim is expecting from us. It needs to run under system python2 where the ROS libraries are accessible. On the other hand our solution uses python3 inside virtual environment and that is where the `subprocess` comes in. 

> Does it test CUDA, or ROS.

This file does not test anything but it runs `cudatest.py` under `python3` which is the actual test of pytorch and tensorflow.

Does it make any sense?

Reply:
Uh ... this is complicated.
Could you add at least some of this as a top-level comment into the script itself?

Reply:
I've added some comments to the script and also to both functions.